package.json

```diff
@@ -267,7 +267,7 @@
 		"playground:build": "npm run storybook:build",
 		"playground:dev": "echo \"Please use storybook:dev instead.\"",
 		"env": "wp-scripts env",
```

Comment:
How about we update `env` instead and remove `wp-env`?

I personally don't see any benefits of running:
`npm run wp-env`
when you can run:
`npx wp-env`

There is also no mention in the documentation of running `env` or `wp-env` scripts.

Reply:
The [Getting Started docs](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/getting-started.md) use: `npm run wp-env` as the example.

I think @noahtallen added it to be specific to ensure you are running the Gutenberg version of wp-env and not a possible old global version that might be installed.

Reply:
I still would prefer `npm run env`, but 🤷

Reply:
this one still points to the old wp-scripts env thing. I think we added a deprecation message to that package, but haven't committed to removing it completely yet?